minseo/controller/postController.js

```diff
@@ -0,0 +1,74 @@
+const postService = require('../service/postService');
+
+const posting = async (req, res) => {
+  try {
+    const { userId, title, content } = req.body;
+    if (!userId || !title || !content) {
+      return res.status(400).json({ message: 'KEY_ERROR' });
+    }
+    await postService.posting(userId, title, content);
```

Comment:
가독성을 위해 구문들 사이에 공백을 추가해 주는 것이 좋습니다.
```suggestion
    const { userId, title, content } = req.body;
    
    if (!userId || !title || !content) {
      return res.status(400).json({ message: 'KEY_ERROR' });
    }
    
    await postService.posting(userId, title, content);
```